### PR TITLE
ENH: Add smriprep-docker wrapper

### PR DESCRIPTION
**Pull request by effigies**

### CODE REVIEW

File: wrapper/smriprep_docker.py [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+import sys
+import os
+import re
+import subprocess
+from warnings import warn
+
+__version__ = '99.99.99'
+__copyright__ = 'Copyright 2019, Center for Reproducible Neuroscience, Stanford University'
+__credits__ = ['Craig Moodie', 'Ross Blair', 'Oscar Esteban', 'Chris Gorgolewski',
+               'Shoshana Berleant', 'Christopher J. Markiewicz', 'Russell A. Poldrack']
```
Comment (effigies) [inline wrapper/smriprep_docker.py]: I did not update this...

Comment (oesteban) [reply wrapper/smriprep_docker.py]: Why not?

Comment (effigies) [reply wrapper/smriprep_docker.py]: Um, mostly because I forgot, then made a note to myself, then forgot again.

File: wrapper/smriprep_docker.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+             ' at https://surfer.nmr.mgh.harvard.edu/registration.html')
+    g_wrap.add_argument(
+        '--use-plugin', metavar='PATH', action='store', default=None,
+        type=os.path.abspath, help='nipype plugin configuration file')
+
+    # Developer patch/shell options
+    g_dev = parser.add_argument_group(
+        'Developer options',
+        'Tools for testing and debugging sMRIPrep')
+    g_dev.add_argument('-f', '--patch-smriprep', metavar='PATH',
```
Comment (effigies) [inline wrapper/smriprep_docker.py]:
```suggestion
    g_dev.add_argument('-s', '--patch-smriprep', metavar='PATH',
```

**Comment by effigies** [comment]
Pretty sure the doc build has been fixed. I'm going to go ahead and merge this. If the build is still broken, I'll open a new PR.

**Pull request merged**